src/commands/validate.ts

```diff
@@ -3,36 +3,31 @@ import * as parser from '@asyncapi/parser';
 import Command from '../base';
 import { ValidationError } from '../errors/validation-error';
 import { load } from '../models/SpecificationFile';
-import { SpecificationFileNotFound } from '../errors/specification-file';
 
 export default class Validate extends Command {
   static description = 'validate asyncapi file';
 
   static flags = {
-    help: flags.help({ char: 'h' })
+    help: flags.help({ char: 'h' }),
+    file: flags.boolean({ char: 'f', description: 'Input will be treated as a file path' }),
```

Comment:
why do we have file, context and user as flags now?

Reply:
I was playing with something, so it was like `asyncapi validate filepath` and then we figure out if that is file or context or URL now I was just adding a flag that specifically tells what that input should be treated at so if you do `asyncapi validate <some-input> -f` then it will take the input as a file

Reply:
I did this to help other commands which are using flags to take file input. Basically `load` function reads the specification from file, context or URL. So even if a command is using the `-f` flag to load the specification file it still does all the checks, which it shouldn't as we are telling it that it is a file. 

I thought that we can use this to solve the issue but I think this should not be in this PR. Actually, I forgot to remove it.

Reply:
yeah, it just causes confusion, just remove them from this pr